### PR TITLE
Clean up some rough edges before our first proper release

### PR DESCRIPTION
**Pull request by nshahquinn**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,19 +1,25 @@
...
+description = (
+  "Tools for analyzing data on SWAP, a platform for confidential "
+  "Wikimedia data"
+)
+
 setup(
     name="wmfdata",
     version=metadata["version"],
-    description="Tools for analyzing data on SWAP, a platform for confidential Wikimedia data",
+    description=description,
```
Comment (bearloga) [inline setup.py]: What is gained by separating this out into a variable?

Comment (nshahquinn) [reply setup.py]: I was reducing the line length, but of course I can just put the parenthetical statement in-line. Done.

**Comment by nshahquinn** [comment]
@bearloga sorry for piling more commits on! I started thinking about how, semantically, our upcoming release should probably be 1.0, and I decided to remove some deprecated bits and do a little more cleanup. I think I'm really done now! 😅

I also did a fairly comprehensive series of tests on the run functions and hive.load_csv

**Review by bearloga — APPROVED** [review]
LGTM

**Pull request merged**